web/components/atomic/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button as AntButton, ButtonProps as AntButtonProps } from 'antd';
+import { FC } from 'react';
+
+import styles from './Button.module.scss';
+
+export type ButtonProps = Omit<AntButtonProps, 'children' | 'icon' | 'title'> & {
+  text: string;
+  iconSrc?: string;
+  iconAltText?: string;
```

Comment:
I think color is missed here, and then not used, so that's why the background color is getting lost.